includes/Media.php

```diff
@@ -117,13 +119,18 @@ public static function init() {
 	 * @return string[] Images.
 	 */
 	public static function get_story_meta_images( $post = null ) {
-		$thumbnail_id = get_post_thumbnail_id( $post );
+		$thumbnail_id = (int) get_post_thumbnail_id( $post );
+
+		if ( ! (bool) $thumbnail_id ) {
```

Comment:
@swissspidy Isn't it `0 === $thumbnail_id` ?

Reply:
Yeah that seems less error prone 👍